pinot-core/src/main/java/com/linkedin/pinot/core/realtime/impl/dictionary/DoubleOffHeapMutableDictionary.java

```diff
@@ -32,7 +32,8 @@
   public DoubleOffHeapMutableDictionary(int estimatedCardinality, int maxOverflowSize) {
     super(estimatedCardinality, maxOverflowSize);
     final int initialEntryCount = nearestPowerOf2(estimatedCardinality);
-    _dictIdToValue = new FixedByteSingleColumnSingleValueReaderWriter(initialEntryCount, V1Constants.Numbers.DOUBLE_SIZE);
+    _dictIdToValue = new FixedByteSingleColumnSingleValueReaderWriter(initialEntryCount, V1Constants.Numbers.DOUBLE_SIZE,
+        null);
```

Comment:
You should probably have a description in the constructor for DoubleOffHeapMutableDictionary so that you can pass this in to FixedByteSingleColumnSingleValueReaderWriter